### ISSUE TITLE
attributes.attributes is missing

### ISSUE THREAD

**Issue by kkamkou** [wopian/kitsu#137]
request: `/example/id?include=something`
plain response:
```
    "included": [
        {
            "id": "...",
            "type": "...",
            "attributes": {attributes: {}, ...} // (!) is also named as "attributes"
        }
    ]
```
Problem: `obj.something.attributes` is missing. Renaming `attributes` to `props` helps. Something eats the `attributes` property.

**Comment by wopian** [comment]
Attributes are hoisted into the resource (`data.attributes.name` -> `data.name`). So `data.attributes.attributes.name` becomes `data.attributes.name`. However the deattribution is a single pass, so the newly added properties from `attributes.attributes` are lost when `data.attributes` is deleted at the end of the `forEach`

Fixing this by deleting `attributes.name` after hoisting and only deleting `data.attributes` if its empty only works on resources within `data`, or the first time an included resource is parsed. If its included by multiple resources, only the first resource has the contents from `attributes.attributes`

**Comment by wopian** [comment]
Fixed in 9.0.2
